build.sbt

```diff
@@ -81,18 +81,22 @@ val customMergeStrategy: String => MergeStrategy = {
   case _ => MergeStrategy.deduplicate
 }
 
-mergeStrategy in assembly := customMergeStrategy
-
-test in assembly := {}
-
 val integrationTest = inputKey[Unit]("Runs the Agora integration tests.")
 
+val unitTest = inputKey[Unit]("Runs the Agora unit tests.")
+
 integrationTest := {
   (testOnly in Test).toTask(" org.broadinstitute.dsde.agora.server.AgoraIntegrationTestSuite").value
 }
 
-val unitTest = inputKey[Unit]("Runs the Agora unit tests.")
-
 unitTest := {
   (testOnly in Test).toTask(" org.broadinstitute.dsde.agora.server.AgoraUnitTestSuite").value
-}
\ No newline at end of file
+}
+
+mergeStrategy in assembly := customMergeStrategy
+
+fork in Test:= true
+
+javaOptions in Test ++= Seq("-Dconfig.file=" + Option(System.getenv("TEST_CONFIG")).getOrElse("src/test/resources/reference.conf"))
```

Comment:
this is mainly just to allow us to inject another conf file... otherwise it just falls back on the previous behavior